File: internal/content/testdata/bench/go/max-hourglass-sum.go
func max_hourglass_sum(grid [][]int) int {
	best := -1<<63
	for i := 0; i < 4; i++ {
		for j := 0; j < 4; j++ {
			s := grid[i][j] + grid[i][j+1] + grid[i][j+2] +
				grid[i+1][j+1] +
				grid[i+2][j] + grid[i+2][j+1] + grid[i+2][j+2]
			if s > best {
				best = s
			}
		}
	}
	return best
}

File: internal/content/testdata/bench/rust/max-hourglass-sum.rs
fn max_hourglass_sum(grid: Vec<Vec<i64>>) -> i64 {
    let mut best = i64::MIN;
    for i in 0..4 {
        for j in 0..4 {
            let s = grid[i][j] + grid[i][j + 1] + grid[i][j + 2]
                + grid[i + 1][j + 1]
                + grid[i + 2][j] + grid[i + 2][j + 1] + grid[i + 2][j + 2];
            if s > best {
                best = s;
            }
        }
    }
    best
}
